routes/users.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const authUser = require('../services/auth/authUser');
+
+const router = express.Router();
+const jsonParser = bodyParser.json();
+const cookieLifeSpan = process.env.JWT_exp; // minutes
```

Comment:
It's generally convention to do const casing for env vars, like so: `JWT_EXPIRATION`